src/hooks/analytics/useTrackNewAccountAdded.ts

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { usePostHog } from 'posthog-react-native';
+import { Account, Web3AuthAccount } from 'types/account';
+import useGetProfile from 'hooks/profile/useGetProfile';
+import { WalletType } from 'types/wallet';
+
+const ACCOUNT_CREATED_EVENT = 'Wallet Created';
+const ACCOUNT_IMPORTED_EVENT = 'Wallet Imported';
+
+/**
+ * Hook to track that a new account has been added from the user.
+ * @param isImported - Tells if the account has been imported or if is
+ * a new one that has been created from the user.
+ */
+const useTrackNewAccountAdded = (isImported: boolean) => {
+  const postHog = usePostHog();
+  const getProfile = useGetProfile();
+
+  return React.useCallback(
+    async (account: Account) => {
+      if (!postHog) {
+        return;
+      }
+
+      const properties: Record<string, any> = {};
+
+      if (!isImported) {
```

Comment:
@bragaz This is what the application track when an account has been imported/created.